<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Einstellungen</class>
 <widget class="QWidget" name="Einstellungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_screenShot_y">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_laserPosX">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_screenShot_h">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_screenShot_x">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_laserPosY">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showLaserSpot">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>show laserSpot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_screenShot_w">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
